README.md

```diff
@@ -111,7 +111,7 @@ bootstrap is the focus of the book *Davison and Hinkley (1997):
 [Bootstrap Methods and Their Application](http://statwww.epfl.ch/davison/BMA/)*.
 Most of the methodology covered in the book is implemented in the
 [boot](http://cran.r-project.org/web/packages/boot/index.html) package for the
-[R programming language](http://www.r-project.org/). [More references](doc/references.md)
+[R programming language](http://www.r-project.org/). [More references](https://github.com/juliangehring/Bootstrap.jl/blob/master/docs/src/references.md)
```

Comment:
You mean if the repo name changes? I made it absolute because people could be accessing the readme locally, but you're probably right that most people see it through github.